{
  "uid" : "953c3292cad08aa5",
  "name" : "Граничные значения у поля \"Пароль\" на странице регистрации",
  "fullName" : "test.test_auth.TestNegativeScenario#test_password_field_boundary_values",
  "historyId" : "abb227afeb648663cd347fd02196aa22",
  "time" : {
    "start" : 1750734768083,
    "stop" : 1750734768083,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nStacktrace:\n#0 0x55a6c1a8ac9a <unknown>\n#1 0x55a6c15306e0 <unknown>\n#2 0x55a6c156b0b2 <unknown>\n#3 0x55a6c156624f <unknown>\n#4 0x55a6c15b68b6 <unknown>\n#5 0x55a6c15b5f76 <unknown>\n#6 0x55a6c15a7c03 <unknown>\n#7 0x55a6c157447b <unknown>\n#8 0x55a6c15750e1 <unknown>\n#9 0x55a6c1a4f44b <unknown>\n#10 0x55a6c1a5337f <unknown>\n#11 0x55a6c1a36f89 <unknown>\n#12 0x55a6c1a53f18 <unknown>\n#13 0x55a6c1a1b6df <unknown>\n#14 0x55a6c1a78308 <unknown>\n#15 0x55a6c1a784e6 <unknown>\n#16 0x55a6c1a89b76 <unknown>\n#17 0x7fc45909caa4 <unknown>\n#18 0x7fc459129c3c <unknown>",
  "statusTrace" : "self = <test.test_auth.TestNegativeScenario object at 0x7f7796ba7e00>\n\n    @pytest.fixture(scope=\"class\")\n    def browser(self) -> Generator[Niffler, Any, None]:\n        options = Options()\n        options.add_argument(\"--lang=en\")\n        # options.add_argument(\"--headless\")\n        options.add_argument(\"--incognito\")\n        options.add_argument(\"--disable-dev-shm-usage\")\n>       wd = webdriver.Chrome(options=options)\n\ntest/test_auth.py:69: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/chrome/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/chromium/webdriver.py:66: in __init__\n    super().__init__(command_executor=executor, options=options)\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:250: in __init__\n    self.start_session(capabilities)\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:342: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:429: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f77961dc6e0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"session not created\",\"message\":\"session not created: probably user data d...\\\\n#16 0x55a6c1a89b76 \\\\u003Cunknown>\\\\n#17 0x7fc45909caa4 \\\\u003Cunknown>\\\\n#18 0x7fc459129c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nE       Stacktrace:\nE       #0 0x55a6c1a8ac9a <unknown>\nE       #1 0x55a6c15306e0 <unknown>\nE       #2 0x55a6c156b0b2 <unknown>\nE       #3 0x55a6c156624f <unknown>\nE       #4 0x55a6c15b68b6 <unknown>\nE       #5 0x55a6c15b5f76 <unknown>\nE       #6 0x55a6c15a7c03 <unknown>\nE       #7 0x55a6c157447b <unknown>\nE       #8 0x55a6c15750e1 <unknown>\nE       #9 0x55a6c1a4f44b <unknown>\nE       #10 0x55a6c1a5337f <unknown>\nE       #11 0x55a6c1a36f89 <unknown>\nE       #12 0x55a6c1a53f18 <unknown>\nE       #13 0x55a6c1a1b6df <unknown>\nE       #14 0x55a6c1a78308 <unknown>\nE       #15 0x55a6c1a784e6 <unknown>\nE       #16 0x55a6c1a89b76 <unknown>\nE       #17 0x7fc45909caa4 <unknown>\nE       #18 0x7fc459129c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: SessionNotCreatedException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "[S]  Session Faker",
    "time" : {
      "start" : 1750734763200,
      "stop" : 1750734763202,
      "duration" : 2
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "[S] App User",
    "time" : {
      "start" : 1750734763202,
      "stop" : 1750734764051,
      "duration" : 849
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Создать пользователя 'qa_guru'",
      "time" : {
        "start" : 1750734763203,
        "stop" : 1750734764050,
        "duration" : 847
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "user_name",
        "value" : "'qa_guru'"
      }, {
        "name" : "user_password",
        "value" : "'qa_guru'"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "[S] Envs",
    "time" : {
      "start" : 1750734763202,
      "stop" : 1750734763202,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "[C] Browser",
    "time" : {
      "start" : 1750734768084,
      "stop" : 1750734769775,
      "duration" : 1691
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nStacktrace:\n#0 0x55a6c1a8ac9a <unknown>\n#1 0x55a6c15306e0 <unknown>\n#2 0x55a6c156b0b2 <unknown>\n#3 0x55a6c156624f <unknown>\n#4 0x55a6c15b68b6 <unknown>\n#5 0x55a6c15b5f76 <unknown>\n#6 0x55a6c15a7c03 <unknown>\n#7 0x55a6c157447b <unknown>\n#8 0x55a6c15750e1 <unknown>\n#9 0x55a6c1a4f44b <unknown>\n#10 0x55a6c1a5337f <unknown>\n#11 0x55a6c1a36f89 <unknown>\n#12 0x55a6c1a53f18 <unknown>\n#13 0x55a6c1a1b6df <unknown>\n#14 0x55a6c1a78308 <unknown>\n#15 0x55a6c1a784e6 <unknown>\n#16 0x55a6c1a89b76 <unknown>\n#17 0x7fc45909caa4 <unknown>\n#18 0x7fc459129c3c <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/niffler-py-st3/niffler-py-st3/python_test/test/test_auth.py\", line 69, in browser\n    wd = webdriver.Chrome(options=options)\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n    ~~~~~~~~~~~~~~~~^\n        browser_name=DesiredCapabilities.CHROME[\"browserName\"],\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n        keep_alive=keep_alive,\n        ^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/chromium/webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 250, in __init__\n    self.start_session(capabilities)\n    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 342, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Регистрация пользователя"
  }, {
    "name" : "story",
    "value" : "Негативные сценарии"
  }, {
    "name" : "epic",
    "value" : "Приложение Niffler"
  }, {
    "name" : "host",
    "value" : "fv-az1372-452"
  }, {
    "name" : "thread",
    "value" : "6871-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_auth"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "length",
    "value" : "13"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 1,
        "skipped" : 0,
        "passed" : 9,
        "unknown" : 0,
        "total" : 10
      },
      "items" : [ {
        "uid" : "149a0a2b71d574df",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/39/#testresult/149a0a2b71d574df",
        "status" : "passed",
        "time" : {
          "start" : 1750733810040,
          "stop" : 1750733810903,
          "duration" : 863
        }
      }, {
        "uid" : "f78dc54562c59b93",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/38/#testresult/f78dc54562c59b93",
        "status" : "passed",
        "time" : {
          "start" : 1750733437024,
          "stop" : 1750733437449,
          "duration" : 425
        }
      }, {
        "uid" : "5984292f80f08403",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/37/#testresult/5984292f80f08403",
        "status" : "passed",
        "time" : {
          "start" : 1750677127271,
          "stop" : 1750677128129,
          "duration" : 858
        }
      }, {
        "uid" : "c0b4f55d5a2749f1",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/36/#testresult/c0b4f55d5a2749f1",
        "status" : "passed",
        "time" : {
          "start" : 1750676762513,
          "stop" : 1750676763611,
          "duration" : 1098
        }
      }, {
        "uid" : "73b897254620f2f1",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/35/#testresult/73b897254620f2f1",
        "status" : "passed",
        "time" : {
          "start" : 1750672834224,
          "stop" : 1750672834883,
          "duration" : 659
        }
      }, {
        "uid" : "d1ef298045458693",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/34/#testresult/d1ef298045458693",
        "status" : "passed",
        "time" : {
          "start" : 1750672137517,
          "stop" : 1750672138795,
          "duration" : 1278
        }
      }, {
        "uid" : "5d38998cad8595c3",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/33/#testresult/5d38998cad8595c3",
        "status" : "passed",
        "time" : {
          "start" : 1750671176149,
          "stop" : 1750671176934,
          "duration" : 785
        }
      }, {
        "uid" : "95ca00992f965867",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/32/#testresult/95ca00992f965867",
        "status" : "passed",
        "time" : {
          "start" : 1750670906625,
          "stop" : 1750670908096,
          "duration" : 1471
        }
      }, {
        "uid" : "b134c6e3bd27a0c3",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/31/#testresult/b134c6e3bd27a0c3",
        "status" : "passed",
        "time" : {
          "start" : 1750670595224,
          "stop" : 1750670596313,
          "duration" : 1089
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "953c3292cad08aa5.json",
  "parameterValues" : [ "13" ]
}